public/app/features/dashboard/specs/exporter.jest.ts

```diff
@@ -0,0 +1,194 @@
+jest.mock('app/core/store', () => {
+  return {
+    getBool: jest.fn(),
+  };
+});
+
+import _ from 'lodash';
+import config from 'app/core/config';
+import { DashboardExporter } from '../export/exporter';
+import { DashboardModel } from '../dashboard_model';
+
+describe('given dashboard with repeated panels', () => {
+  var dash, exported;
+
+  beforeEach(done => {
+    dash = {
+      templating: { list: [] },
+      annotations: { list: [] },
+    };
+
+    config.buildInfo = {
+      version: '3.0.2',
+    };
+
+    dash.templating.list.push({
+      name: 'apps',
+      type: 'query',
+      datasource: 'gfdb',
+      current: { value: 'Asd', text: 'Asd' },
+      options: [{ value: 'Asd', text: 'Asd' }],
+    });
+
+    dash.templating.list.push({
+      name: 'prefix',
+      type: 'constant',
+      current: { value: 'collectd', text: 'collectd' },
+      options: [],
+    });
+
+    dash.templating.list.push({
```

Comment:
Instead of using `dash.templating.list.push`, `dash.annotations.list.push` and `dash.panels =` and `dash.panels.push` you could initiate everything above in line 16 (`dash = {`)